.eslintrc.js

```diff
@@ -47,13 +47,15 @@ module.exports = {
       files: ['test/**/*'],
       rules: {
         'import/no-extraneous-dependencies': 'off',
+        'import/no-named-as-default': 'off',
       },
     },
     {
       files: ['**/__tests__/**'],
       rules: {
         'functional/no-expression-statement': ['off'],
         'import/no-extraneous-dependencies': ['off'],
+        'import/no-named-as-default': 'off',
```

Comment:
Fixing the project references and including `project: ['./']` in `.eslint.js` (so it could find the tsconfig paths) must have let the `eslint-plugin-import` better analyze the import statements, because it started failing this import.

```js
import Big from 'big.js`;
```

The rule `import/no-named-as-default` prefers:

```js
import { Big } from 'big.js';
```

So we can change it or ignore it.

Reply:
I'd prefer fixing the imports and keeping the rule on. If there are imports that can't be fixed, we can ignore the rule locally.

Reply:
I will change it.

Reply:
@johnhooks This should be the last change before we can merge.

Reply:
I don't know why I periodically had this error happening. But I'll look locally for the solution because it isn't in CI.

Reply:
It is hard jumping between this and #667, I hadn't removed the rule as you suggested. THAT made the error come back lol.